share/completions/argparse.fish

```diff
@@ -0,0 +1,77 @@
+function __argparse_exclusive_generate_args --description 'Helper function to list unused options'
+  set --local all_tokens (commandline --tokenize)
```

Comment:
We use 4 space indentation - you can run `fish_indent` on the fish scripts to reformat them